### ISSUE TITLE
installation

### ISSUE THREAD

**Issue by hristodelev1** [3fon3fonov/exostriker#104]
I try to install with the pip install command and the following message appears. 

 Cannot unpack file /tmp/pip-bjHddL-unpack/exostriker (downloaded from /tmp/pip-bF4u0p-build, content-type: text/html; charset=utf-8); cannot detect archive format
Cannot determine archive format of /tmp/pip-bF4u0p-build

I also try to install it with the git clone command and a message saying "no module named Pathos".

**Comment by 3fon3fonov** [comment]
Hi Hristo!

I have just updated https://github.com/3fon3fonov/exostriker/blob/exostriker-ready/README.md


Please check the instructions below "Installation", and make sure you follow the installation pattern correctly. Please, let me know if you keep having problems. I would need to know your Python3 version, OS, and how you are trying to install the tool.
Thanks!

**Comment by hristodelev1** [comment]
Hello,
currently, I am using Ubuntu 18.04 and following the instructions have installed Python 3.8.  I tried again to install using the pip install method and the following message appeared "   ERROR: Cannot unpack file /tmp/pip-unpack-g0unpviv/exostriker (downloaded from /tmp/pip-req-build-b1mx9htg, content-type: text/html; charset=utf-8); cannot detect archive format
ERROR: Cannot determine archive format of /tmp/pip-req-build-b1mx9htg" which is different than the previous one. 

I tried the git clone method" python 3.8 exostriker_gui.py install"  and the same " no module named Pathos" appears.

Thank you!
